### DISCUSSION TITLE
PKI OCSP Plugins OpenXKPI

### DISCUSSION THREAD

**Discussion by Flavinux** [strongswan/strongswan#2513] [Q&A] [answered]
Hello, 
i'm looking for help with PKI and OpenXPKI Pluggins, Compiled from source with --enable-openxpki and --enable-mysql. I didn't arrive to setup correctly the database URI cf https://docs.strongswan.org/docs/5.9/plugins/openxpki.html "openxpki database URI missing" /etc/strongswan.conf https://pastebin.com/qsC4bnQr , 
thanks

**Comment by tobiasbrunner** [comment, marked as answer]
You need to configure the database URI in the `pki` section (or the `libstrongswan` section so it applies globally), not the `charon` section, where it only applies to the charon daemon (which won't use that plugin at all).
